documentation/resource_atlantis_installer.md

```diff
@@ -0,0 +1,22 @@
+# atlantis_installer
+
```

Comment:
Can you please list the actions similar to how we do it [elsewhere](https://github.com/sous-chefs/keepalived/blob/master/documentation/keepalived_global_defs.md#actions)? Please do it globally and also document which one is the default.